repos/system_upgrade/el7toel8/actors/repositoriesblacklist/actor.py

```diff
@@ -9,13 +9,21 @@ class RepositoriesBlacklist(Actor):
     """
 
     name = 'repositories_blacklist'
-    consumes = ()
+    consumes = (RepositoriesFacts,)
     produces = (RepositoriesBlacklisted,)
     tags = (IPUWorkflowTag, FactsPhaseTag)
 
+    def _is_repo_enabled(self, repoid):
```

Comment:
shouldn't this function be moved to library?

Reply:
Not necessary. The actor can be tested as whole component. I am thinking more about shared library to work with repositories completely but this is for following PR.